### PR TITLE
Update REAMDE.md

### PR DESCRIPTION
**Pull request by Lord-Haji**
Add Citra discord url and Citra forums url directly in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,9 @@ Citra is an experimental open-source Nintendo 3DS emulator/debugger written in C
 
 Citra is licensed under the GPLv2 (or any later version). Refer to the license.txt file included. Please read the [FAQ](https://github.com/citra-emu/citra/wiki/FAQ) before getting started with the project.
 
-Check out our [website](https://citra-emu.org/)!
+Check out our [website](https://citra-emu.org/)!<br />
```
Comment (Dragios) [inline README.md]: Check out our official [website](https://citra-emu.org/) for more information and updates regarding Citra emulator.

File: README.md [hunk 2]
```diff
@@ -9,7 +9,9 @@ Citra is an experimental open-source Nintendo 3DS emulator/debugger written in C
...
+For more help check out our [forums](https://discuss.citra-emu.org/)!<br />
```
Comment (Dragios) [inline README.md]: For those seeking for helps and advice kindly visit our [forums](https://discuss.citra-emu.org/) to do so.

File: README.md [hunk 3]
```diff
@@ -9,7 +9,9 @@ Citra is an experimental open-source Nintendo 3DS emulator/debugger written in C
...
+For talking to other Citra lovers join our [Discord Chat](https://discordapp.com/invite/fZwvKPu)!
```
Comment (Dragios) [inline README.md]: Want to have a chance to chat with other Citra members or even developers? Join our [Discord channel](https://discordapp.com/invite/fZwvKPu) which is open for everyone.

Lovers/Members/Users or etc. Not sure which to decide. This is a subjective matter.

File: README.md (hunk 2 above)
Comment (Lord-Haji) [inline README.md]: isn't it kinda wrong english

File: README.md (hunk 1 above)
Comment (Lord-Haji) [inline README.md]: actually I am new to this was just using <br> tag to go to next line

**Comment by shinyquagsire23** [comment]
Not sure why there's text as part of the image, would probably be best to just have the image be the Citra logo and also the image as part of the repo or Citra's website rather than a random imgur link. On top of that, I'm fairly sure there's a proper formatting for images besides html, ie
```
![caption](image url)
```

**Comment by jroweboy** [comment]
I vote to leave out the HTML entirely. While github does(!) render it, its very surprising to see HTML in a markdown document. Replace the `<br />` with just a regular newline and replace the html image with what shiny quagsire posted. 

Also, the citra logo is (slowly) changing to the new logo on the website. https://citra-emu.org/theme/citra-bs/images/logo.png is the new logo png. There's also a svg version of it somewhere. Add it to the repository in the dist directory and use that as the image url in the Readme

Other than that, I'm fine with the idea behind the changes.

**Comment by Lord-Haji** [comment]
Ok ill do it tomorrow

**Comment by wwylele** [comment]
What's the status of this PR?

File: README.md [hunk 4]
```diff
@@ -1,3 +1,5 @@
+<p align="center"><img src="http://i.imgur.com/yTt3XWm.png" alt="Citra Emulator"></p>
```
Comment (JayFoxRox) [inline README.md]: This is our old logo. I don't think we use it anymore

File: README.md [hunk 5]
```diff
@@ -9,9 +11,11 @@ Citra is an experimental open-source Nintendo 3DS emulator/debugger written in C
 
 Citra is licensed under the GPLv2 (or any later version). Refer to the license.txt file included. Please read the [FAQ](https://github.com/citra-emu/citra/wiki/FAQ) before getting started with the project.
 
-Check out our [website](https://citra-emu.org/)!
+→Check out our official [website](https://citra-emu.org/)! For more information and updates regarding Citra emulator<br /><br />
```
Comment (JayFoxRox) [inline README.md]: These arrows look bad - If you want a list use markdown lists, same about the HTML breaks

**Comment by JayFoxRox** [comment]
This has been more than a month old for really unimportant changes.
I like the idea of these changes, but the current implementation is pretty bad.

I suggest you fix all these issues and squash these, or we'll have to close this PR.
PRs are piling up and we'll have to get rid of some

**Comment by Lord-Haji** [comment]
Ill squash alll the issues in in PR for now closing it for convinience...
I was having exams....

**Comment by JayFoxRox** [comment]
Please open a new PR once you are ready. Closed.

**Pull request closed without merging**